website/content/docs/concepts/workers.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Workers
+description: |-
+  Introduction to Boundary workers
+---
+
+# Workers
+
+## Overview
+Boundary's architecture consists of 2 main components:
+1. **Control Plane** - made up of controllers
+2. **Data Plane** - made up of workers
+3. **Client** - installed on the user's device
+
+**Controllers** are what users authenticate to through the client, and contain Boundary's resources and permissions. In addition, controllers also communicate with external components 
+such as the database, KMS, Vault, identity providers, and plugins. **Workers** are primarily used as network proxies for Boundary sessions, allowing users to access 
+private targets. Instead of exposing a private network to the public, or allowing users to have access to entire private networks, workers create a direct network 
+tunnel between users and targets. 
+
+![IAM Example Structure](/img/access-model.png)
+
+## Use cases
+**Session proxying**
+
+Workers can be used to proxy sessions between clients and targets located in public or private networks. In addition, workers can perform 
+[multi-hop](#multi-hop-sessions-hcp-ent) sessions, where workers form a chain of proxies to reach deeper into protected networks.
+
+**Worker authentication**
+
+Workers that are not directly connected to controllers can communicate through intermediary workers to register to the control plane.
```

Comment:
Might be worth clarifying here that this is also multi-hop, and touching on how workers can auth in single-hop scenarios/ linking to https://developer.hashicorp.com/boundary/docs/concepts/security/connections-tls#client-to-worker-tls

Reply:
Will do